server/plugin.go

```diff
@@ -150,13 +150,13 @@ func (p *Plugin) completeAuthorizeInADD(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	userId, err := p.API.KVGet(state)
+	userID, err := p.API.KVGet(state)
 
 	if err != nil {
 		fmt.Println(err.Message)
 		http.Error(w, "cannot get stored state", http.StatusBadRequest)
 		return
-	} else if userId == nil {
+	} else if userID == nil {
```

Comment:
you can drop the `else` here. Would you mind applying this change to the whole file?

Reply:
Will do.